zubhub_frontend/zubhub/src/views/login/Login.jsx

```diff
@@ -63,10 +63,12 @@ function Login(props) {
   const { show_password } = state;
   const { t } = props;
 
+  const styleOverridesPresent = Object.keys(props?.styleOverrides).length > 0
```

Comment:
This is not a nice idea to get keys on each render.

try classNames prop for different elements you want to override. For example of Checkbox component
```javascript
const Checkbox = (props) => {
  const classes = useStyles();
  return (
    <div className={clsx([classes.container, props.classNames?.container])}>
      <input type="checkbox" className={clsx([classes.checkbox, props.classNames?.checkbox])} />
    </div>
  )
}
```

If I want to override styles I could do something like

```javascript

const RedCheckbox = () => {
  const classes = useStyles();
  return <Checkbox classNames={{ checkbox: classes.redAccentCheckbox }} />
}
```